src/components/Week/Week.jsx

```diff
@@ -1,87 +1,105 @@
-        <Grid container spacing={{ xs: 2, md: 3 }} columns={{ xs: 4, sm: 8, md: 12 }}>
+        <Box sx={{flexGrow: 1}}>
+            <Grid container spacing={{xs: 2, md: 3}}>
 
-          <Grid key={1} size={{ xs: 2, sm: 4, md: 6 }}>
-            <Card sx={{ maxWidth: 345 }}>
-              <CardActionArea>
-              <WbSunnyIcon />
-                <CardContent>
-                  <Typography gutterBottom variant="h5" component="div">
-                    Bariloche
-                  </Typography>
-                  <Typography variant="body2" sx={{ color: 'text.secondary' }}>
-                    Sunny windy weather today but we love eating chocolates and hiking in the mountains with a lot of snow and skiing super fast.
-                  </Typography>
-                </CardContent>
-              </CardActionArea>
-            </Card>
-          </Grid>
-          
-          <Grid key={1} size={{ xs: 2, sm: 4, md: 6 }}>
-            <Card sx={{ maxWidth: 345 }}>
-              <CardActionArea>
-              <WbSunnyIcon />
-                <CardContent>
-                  <Typography gutterBottom variant="h5" component="div">
-                    Moron
-                  </Typography>
-                  <Typography variant="body2" sx={{ color: 'text.secondary' }}>
-                    Sunny windy weather today. 
-                  </Typography>
-                </CardContent>
-              </CardActionArea>
-            </Card>
-          </Grid>
-          
-          <Grid key={1} size={{ xs: 2, sm: 4, md: 6 }}>
-          <Card sx={{ maxWidth: 345 }}>
-              <CardActionArea>
-              <ThunderstormIcon fontSize='large' htmlColor='blue'/>
-                <CardContent>
-                  <Typography gutterBottom variant="h5" component="div">
-                    Tandil
-                  </Typography>
-                  <Typography variant="body2" sx={{ color: 'text.secondary' }}>
-                    The forecast for Tandil city shows very have storms with possibility of tornado, earthquake
-                  </Typography>
-                </CardContent>
-              </CardActionArea>
-            </Card>
-          </Grid>
-          <Grid key={1} size={{ xs: 2, sm: 4, md: 6, lg: 6 }}>
-          <Card sx={{ maxWidth: 345 }}>
-              <CardActionArea>
-              <AcUnitIcon />
-                <CardContent>
-                  <Typography gutterBottom variant="h5" component="div">
-                    Mar del Plata
-                  </Typography>
-                  <Typography variant="body2" sx={{ color: 'text.secondary' }}>
-                    Mardel people are a widespread group of squamate reptiles, with over 6,000
-                    species, ranging across all streets except Champagnat
-                  </Typography>
-                </CardContent>
-              </CardActionArea>
-            </Card>
-          </Grid>
+                <Grid item key={1} size={{xs: 12, sm: 12, md: 3}}>
```

Comment:
I fixed indentation and replaced card component with paper to get the elevated surface effect.